Benchmarks/Benchmarks/ProtocolBenchmark/ProtocolBenchmark.swift

```diff
@@ -0,0 +1,46 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the RediStack open source project
+//
+// Copyright (c) 2023 RediStack project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of RediStack project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@_spi(RESP3) import RediStack
+import Benchmark
+import NIOCore
+
+let benchmarks = {
+    let resp3ArrayValueBuffer = ByteBuffer(string: "*2\r\n$3\r\nGET\r\n$7\r\nwelcome\r\n")
+    let resp3ArrayCount = 2
+
+    Benchmark("RESP3 Array Parsing") { benchmark in
```

Comment:
```suggestion
    Benchmark("RESP3 Array Parsing", configuration: .init(scalingFactor: .kilo)) { benchmark in
       for _ in benchmark.scaledIterations {
        try runRESP3ArrayParsing(
             valueBuffer: resp3ArrayValueBuffer,
             valueCount: resp3ArrayCount
         )
         }
```

Reply:
Would suggest using such scaling factor (.kilo) for the `RESP Array Parsing` and `RESP3 Array Parsing` benchmarks, as they have very short runtimes. The scaling factor is used as an inner loop and the results will be scaled up accordingly for throughput if you update to a recent benchmark version.